tests/phpunit/tests/media.php

```diff
@@ -3162,6 +3162,31 @@ public function test_wp_img_tag_add_loading_attr_opt_out() {
 		$this->assertStringNotContainsString( ' loading=', $img );
 	}
 
+
+
+	/**
+	 * Test that decoding="async" is not applied to img tags with single quotes.
+	 *
+	 * @ticket 56969
+	 */
+	public function test_wp_img_tag_add_decoding_attr_with_single_quotes() {
+		$img = "<img src='example.png' alt=' width='300' height='225' />";
+		$img = wp_img_tag_add_decoding_attr( $img, 'test' );
+
+		$this->assertStringNotContainsString( ' decoding="async"', $img );
+	}
+
+	/**
+	 * Test that decoding="async" os not applied to img tags inside JSON.
+	 *
+	 * @ticket 56969
+	 */
+	public function test_decoding_async_not_applied_to_json() {
+		$content = '{"image": "<img src=\"example.png\" alt=\" width=\"300\" height=\"225\" />"}';
```

Comment:
Same as above.
```suggestion
		$content = '{"image": "<img src=\"example.png\" alt=\"\" width=\"300\" height=\"225\" />"}';
```

Reply:
Another nit: `\` should be escaped as `\\`, even in single quoted strings.

PHP parses it fine as is